streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/config/SpringProperties.java

```diff
@@ -87,17 +87,37 @@ private static void dataSourceConfig(Properties userConfig, Properties springCon
         springConfig.put("spring.datasource.driver-class-name", "org.postgresql.Driver");
         break;
       case "h2":
-        springConfig.put("spring.datasource.driver-class-name", "org.h2.Driver");
-        springConfig.put("spring.datasource.username", "sa");
-        springConfig.put("spring.datasource.password", "sa");
+        String h2Dir = userConfig.getProperty("datasource.h2-data-dir", null);
```

Comment:
Is it better to use `h2DataDir` instead of `h2Dir` which is more intuitive.

Reply:
This parameter is read from config.yaml. h2DataDir naming style has a minor issue.

Reply:
> Is it better to use `h2DataDir` instead of `h2Dir` which is more intuitive.

done